### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Screen Reader: Narrator/NVDA is not reading the labels of the text field and checkbox present on the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#100]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.1

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. verify that screen reader is reading the labels of all the input form fields on the page.

**Actual Result:**
Narrator/NVDA is not reading the label of the "Access Token" text field and "Keep me connected from now on" checkbox.

**Expected Result:**
Screen Reader should read the label of all the input elements on the page.

**User Impact:**
Screen reader users will be unable to know the labels of the form field and hence they will not be able to provide inputs to the form fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug 100_Screen reader is not reading the label of the input elements.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323592/Bug.100_Screen.reader.is.not.reading.the.label.of.the.input.elements.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This UI is not used anymore.

**Comment by Arjun553** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS1.3.1;#Narrator: #NVDA; #Screen Reader; #.NETWebsites; #Edge; #Chrome

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
